src/serac/physics/solid_mechanics.hpp

```diff
@@ -875,7 +875,7 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
    * @note This method must be called prior to completeSetup()
    */
   template <int... active_parameters, typename MaterialType, typename StateType = Empty>
-  void setMaterial(DependsOn<active_parameters...>, MaterialType&& material, qdata_type<StateType> qdata = EmptyQData)
+  void setMaterial(DependsOn<active_parameters...>, MaterialType material, qdata_type<StateType> qdata = EmptyQData)
   {
     MaterialStressFunctor<MaterialType> material_functor(material, geom_nonlin_);
```

Comment:
This should have been `std::forward<MaterialType>(material)` if the forwarding reference was kept on material.

Reply:
Oh wait, there's a call to move later. So here I think you would actually want a copy, and at line 888, you would call std::forward instead of std::move.